engine/src/main/java/org/camunda/bpm/engine/impl/el/ExpressionManager.java

```diff
@@ -16,147 +16,12 @@
  */
 package org.camunda.bpm.engine.impl.el;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
-import org.camunda.bpm.engine.delegate.VariableScope;
-import org.camunda.bpm.engine.impl.core.variable.scope.AbstractVariableScope;
-import org.camunda.bpm.engine.impl.javax.el.ArrayELResolver;
-import org.camunda.bpm.engine.impl.javax.el.CompositeELResolver;
-import org.camunda.bpm.engine.impl.javax.el.ELContext;
-import org.camunda.bpm.engine.impl.javax.el.ELResolver;
-import org.camunda.bpm.engine.impl.javax.el.ExpressionFactory;
-import org.camunda.bpm.engine.impl.javax.el.FunctionMapper;
-import org.camunda.bpm.engine.impl.javax.el.ListELResolver;
-import org.camunda.bpm.engine.impl.javax.el.MapELResolver;
-import org.camunda.bpm.engine.impl.javax.el.ValueExpression;
-import org.camunda.bpm.engine.impl.juel.ExpressionFactoryImpl;
-import org.camunda.bpm.engine.test.mock.MockElResolver;
-import org.camunda.bpm.engine.variable.context.VariableContext;
-
-
 /**
- * <p>
- * Central manager for all expressions.
- * </p>
- * <p>
- * Process parsers will use this to build expression objects that are stored in
- * the process definitions.
- * </p>
- * <p>
- * Then also this class is used as an entry point for runtime evaluation of the
- * expressions.
- * </p>
+ * Original ExpressionManager class is renamed to {@link JuelExpressionManager}
+ * to make el pluggable.
  *
- * @author Tom Baeyens
- * @author Dave Syer
- * @author Frederik Heremans
+ * @author Jan Cheng
```

Comment:
I would suggest leaving the previous class comment as is and just removing the author tags.